widget/embedded/src/utils/wallets.ts

```diff
@@ -551,3 +551,7 @@ export const isFetchingBalance = (
   !!connectedWallets.find(
     (wallet) => wallet.chain === blockchain && wallet.loading
   );
+
+export function stringWalletsState(walletsInfo: ModalWalletInfo[]): string {
```

Comment:
Since TypeScript can infer the return type of this function, we don't need ": string".
Nit: renaming the function can be useful.

Reply:
Fixed